src/drivers/altimu-10/altimu-10.h

```diff
@@ -0,0 +1,146 @@
+#ifndef ALTIMU_10_H
+#define ALTIMU_10_H
+
+#include "rocketlib/include/common.h"
+#include <stdbool.h>
+#include <stdint.h>
+
+// Sensor data structure definitions
+typedef struct {
+    int16_t x;
+    int16_t y;
+    int16_t z;
+} altimu_accelerometer_data;
+
+typedef struct {
+    int16_t x;
+    int16_t y;
+    int16_t z;
+} altimu_gyroscope_data;
+
+typedef struct {
+    int16_t x;
+    int16_t y;
+    int16_t z;
+} altimu_magnetometer_data;
+
+typedef struct {
+    float pressure;
+    float temperature;
+} altimu_barometer_data;
```

Comment:
Do the device registers for pressure and temp store data as a float? Otherwise I assume these would be ints like the IMU data?